src/Xray/XrayJasConfigClient.ts

```diff
@@ -0,0 +1,42 @@
+import { HttpClient, IRequestParams } from '../HttpClient';
+import { IClientResponse, ILogger } from '../../model';
+import { IJasConfig } from '../../model/Xray/JasConfig/JasConfig';
+
+export class XrayJasConfigClient {
+    private readonly jasConfigurationEndpoint: string = '/api/v1/configuration/jas';
+    constructor(private readonly httpClient: HttpClient, private readonly logger: ILogger) {}
+    /**
+     *
+     * Sends 'GET /configuration/js requests to Xray and waits for 200 response'.
+     * @returns the jas config
+     * @throws an exception if an unexpected response received from Xray or if checkCanceled threw an exception
+     */
+    async getJasConfig(): Promise<IJasConfig> {
```

Comment:
no `checkCanceled` func provided. either implement canceling (preferable since we can cancel scans in the IDE while they are running, see `XrayScanClient.ts` for details) or remove comment

Reply:
removed comment